cli/replicaset/roles.go

```diff
@@ -2,17 +2,17 @@ package replicaset
 
 import "fmt"
 
-// RolesAddCtx describes a context for adding roles.
-type RolesAddCtx struct {
-	// InstName is an instance name in which add role.
+// RolesUpdateCtx describes a context for adding/removing roles.
+type RolesUpdateCtx struct {
```

Comment:
Also a good name as an alternative for `RolesChangeCtx`.

Reply:
Edited.